package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "you-dont-need-javascript",
+  "version": "1.0.0",
+  "description": "Please note these demos should be considered as CSS \"Proofs of Concepts\". They may have serious issues from accessibility point of view (keyboard navigation, speech synthesis, etc.), or progressive enhancement/degradation/etc.",
+  "main": "Gruntfile.js",
+  "directories": {
+    "example": "examples"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/you-dont-need/You-Dont-Need-Javascript"
+  },
+  "keywords": [
+    "you",
+    "dont",
+    "need",
+    "javascript"
+  ],
+  "author": "you-dont-need",
+  "license": "ISC",
```

Comment:
Not consistent with [existing license](https://github.com/you-dont-need/You-Dont-Need-JavaScript/blob/master/LICENSE)

Reply:
It's the default of `npm init` (MIT compatible?)

Reply:
Yes MIT compatible but not the same as the existing project license (WTFPL). I would favor seeking consensus before marking this project with a different license.